index.html

```diff
@@ -0,0 +1,1249 @@
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+            </section>
+            <!-- Buttons -->
+
+            <!-- Responsive image -->
+
+            <section id="image">
+                <div class="code-container">
+                    <h2 class="component-title">Responsive Image</h2>
+                    <img class="img-responsive" src="https://source.unsplash.com/random" />

```

Comment:
Normal Image is not responsive it has a fixed width.

Reply:
fixed